<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>261</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>VGG16 TEST</string>
    </property>
    <widget class="QPushButton" name="Show_Train_Images">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>221</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>1. Show Train Images</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Show_HyperParameter">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>221</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>2. Show HyperParameter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Show_Model_Shortcut">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>221</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>3. Show Model Shortcut</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Show_Accuracy">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>221</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>4. Show Accuracy</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Train">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>221</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>6. Train</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>221</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Test</string>
     </property>
     <widget class="QPushButton" name="Test">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>65</y>
        <width>181</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>5. Test</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Test_Result_InputBox">
      <property name="geometry">
       <rect>
        <x>119</x>
        <y>30</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>31</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Index  of  Image:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>302</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
